runtime/lua/vim/treesitter/languagetree.lua

```diff
@@ -78,6 +80,11 @@ function LanguageTree:invalidate(reload)
   end
 end
 
+---@param injections table<string, table> table of lang to list of regions
+function LanguageTree:set_custom_injections(injections)
```

Comment:
Do we need a new function for this? Can't we just pass the `_custom_injections` as part of the `opts` during parser creation?